app/src/main/java/com/nervousfish/nervousfish/activities/BluetoothConnectionActivity.java

```diff
@@ -54,7 +54,6 @@
     // Used to fill the listview of newly discovered Bluetooth devices
     private ArrayAdapter<String> newDevicesArrayAdapter;
     // Used to fill the listview of paired Bluetooth devices
-    private ArrayAdapter<String> pairedDevicesArrayAdapter;
```

Comment:
Can you remove the unused comment "// Used to fill the listview of paired Bluetooth devices"